intermediate_source/torch_compile_tutorial.py

```diff
@@ -33,9 +33,25 @@
 # - ``numpy``
 # - ``scipy``
 # - ``tabulate``
-#
-# Note: a modern NVIDIA GPU (Volta or Ampere) is recommended for this tutorial.
-#
+
+######################################################################
+# NOTE: a modern NVIDIA GPU (A100 or V100) is recommended for this tutorial in
+# order to reproduce the speedup numbers shown below and documented elsewhere.
+
+import torch
+import warnings
+
+gpu_ok = False
+if torch.cuda.is_available():
+    device = torch.cuda.get_device_name()
+    if "V100" in device or "A100" in device:
```

Comment:
Better use device capabilities rather than vendor name
```suggestion
    device_cap = torch.cuda.get_device_capability()
    if device_cap in [(7, 0), (8,0)]:
```